index.html

```diff
@@ -216,9 +216,9 @@ <h3 id="unacceptablebehaviour">
       <li>Retaliating against anyone who files a complaint that someone has violated this code of conduct.</li>
     </ul>
 
-    <p>This Code prioritises the safety of individuals within marginalised communities over the
+    <p>The enforcers of this Code should prioritise the safety of individuals within marginalised communities over the
```

Comment:
I believe @AdaRoseCannon was asked to change this to "this code"

Reply:
We each did in different branches. Anyway, merged now separate from this PR.

Reply:
It doesn't make sense to say that "the Code" ... "reserves the rights not to take further actions". It used to be "the enforcers" who are capable of making decisions - the Code prescribes recommended behavior but doesn't make decision decisions on its own. Rewrite needed.

Reply:
Agreed. Noting that we need to do more work based on #74 ...